test/config/logs.go

```diff
@@ -0,0 +1,44 @@
+package config
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/onsi/ginkgo"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	// Formatter is the format configuration to write logs into text
+	Formatter = logrus.TextFormatter{
+		DisableTimestamp: true,
+	}
+
+	// TestLogWriter buffer to store all the logs generated by the test
+	TestLogWriter bytes.Buffer
+	// TestLogFileName is the default name to store test logs
+	TestLogFileName = "logs"
+)
+
+// LogHook to send logs via `ginkgo.GinkgoWriter`.
+type LogHook struct{}
+
+// Levels defined levels to send logs to FireAction
+func (h *LogHook) Levels() []logrus.Level {
+	return []logrus.Level{
+		logrus.WarnLevel,
+		logrus.ErrorLevel,
+		logrus.FatalLevel,
+		logrus.PanicLevel,
+	}
+}
+
+// Fire is a callback function used by logrus to write logs that matchs in
+// the given by `Levels` method
+func (h *LogHook) Fire(entry *logrus.Entry) (err error) {
+	line, err := Formatter.Format(entry)
+	if err == nil {
+		fmt.Fprintf(ginkgo.GinkgoWriter, string(line))
```

Comment:
Will this buffer to `TestLogWriter` above? or is the idea that we write logrus logs to GinkgoWriter directly but buffer Cmd output (from the SendToLog call in `Exec` (https://github.com/cilium/cilium/pull/2290/files#diff-5a466145a9fd897d3ecef38629d668fdR123)

Reply:
Hello! 

The main idea, `warn, error, fatal, panic` will be reported into GingkoWritter (This write the logs to the console + Junit output), the other ones will be reported to `TestLogWriter` and write it to the test log.